api-client/src/models/LocationModel.ts

```diff
@@ -113,6 +114,13 @@ class LocationModel {
     )
   }
 
+  get coordinates(): Position | null {
+    if (this.longitude == null || this.latitude == null) {
```

Comment:
Please also check for undefined and use strict equal checks (`===`)

Reply:
done